Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json

```diff
@@ -6570,7 +6574,7 @@
 																				"header": [],
 																				"body": {
 																					"mode": "raw",
-																					"raw": "  {\r\n    \"localEducationAgencyReference\": {\r\n      \"localEducationAgencyId\": 255901\r\n    },\r\n    \"schoolId\": 7890,\r\n    \"nameOfInstitution\": \"ABCD Charter School\",\r\n    \"operationalStatusDescriptor\": \"uri://ed-fi.org/OperationalStatusDescriptor#Active\",\r\n    \"shortNameOfInstitution\": \"ABCD\",\r\n    \"webSite\": \"http://www.GBISD.edu/ABCD/\",\r\n    \"administrativeFundingControlDescriptor\": \"uri://ed-fi.org/AdministrativeFundingControlDescriptor#Public School\",\r\n    \"charterStatusDescriptor\": \"uri://ed-fi.org/CharterStatusDescriptor#School Charter\",\r\n    \"schoolTypeDescriptor\": \"uri://ed-fi.org/SchoolTypeDescriptor#Regular\",\r\n    \"titleIPartASchoolDesignationDescriptor\": \"uri://ed-fi.org/TitleIPartASchoolDesignationDescriptor#Not A Title I School\",\r\n    \"addresses\": [],\r\n    \"educationOrganizationCategories\": [\r\n      {\r\n        \"educationOrganizationCategoryDescriptor\": \"uri://ed-fi.org/EducationOrganizationCategoryDescriptor#School\"\r\n      }\r\n    ],\r\n    \"schoolCategories\": [\r\n      {\r\n        \"schoolCategoryDescriptor\": \"uri://ed-fi.org/SchoolCategoryDescriptor#Middle School\"\r\n      }\r\n    ],\r\n    \"gradeLevels\": [\r\n      {\r\n        \"gradeLevelDescriptor\": \"uri://ed-fi.org/GradeLevelDescriptor#Sixth grade\"\r\n      },\r\n      {\r\n        \"gradeLevelDescriptor\": \"uri://ed-fi.org/GradeLevelDescriptor#Seventh grade\"\r\n      },\r\n      {\r\n        \"gradeLevelDescriptor\": \"uri://ed-fi.org/GradeLevelDescriptor#Eighth grade\"\r\n      }\r\n    ]\r\n  }",
+																					"raw": "",
```

Comment:
This is a GET request. Body was apparently not cleared when request was copied from a non-GET request.